Podcast/FollowerFollowingViewController.swift

```diff
@@ -0,0 +1,110 @@
+//
+//  FollowerFollowingViewController.swift
+//  Podcast
+//
+//  Created by Drew Dunne on 9/10/17.
+//  Copyright © 2017 Cornell App Development. All rights reserved.
+//
+
+/**
+ * This view controller is for both displaying your followers and whom you follow.
+ */
+
+import UIKit
+import NVActivityIndicatorView
+
+class FollowerFollowingViewController: UIViewController, UITableViewDataSource, UITableViewDelegate {
+    
+    let cellIdentifier = "searchUsersCell"
+    
+    var usersTableView: UITableView!
+    var loadingActivityIndicator: NVActivityIndicatorView!
+    var refreshControl: UIRefreshControl!
+    
+    var users: [User] = []
+    var followersOrFollowings: UserFollowsType!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .podcastWhiteDark
+        title = followersOrFollowings == .Followers ? "Followers" : "Following"
+
+        guard let appDelegate = UIApplication.shared.delegate as? AppDelegate else { return }
+        // Do any additional setup after loading the view.
+        usersTableView = UITableView(frame: CGRect.zero)
+        usersTableView.contentInset.bottom = appDelegate.tabBarController.tabBarHeight
+        usersTableView.delegate = self
+        usersTableView.dataSource = self
+        usersTableView.backgroundColor = .clear
+        usersTableView.separatorStyle = .none
+        usersTableView.showsVerticalScrollIndicator = false
+        usersTableView.register(SearchPeopleTableViewCell.self, forCellReuseIdentifier: cellIdentifier)
+        usersTableView.reloadData()
+        view.addSubview(usersTableView)
+        
+        loadingActivityIndicator = createLoadingAnimationView()
+        loadingActivityIndicator.center = view.center
+        view.addSubview(loadingActivityIndicator)
+        
+        refreshControl = UIRefreshControl()
+        refreshControl.tintColor = .podcastTeal
+        refreshControl.addTarget(self, action: #selector(FollowerFollowingViewController.handleRefresh), for: UIControlEvents.valueChanged)
+        usersTableView.addSubview(refreshControl)
+        
+        usersTableView.snp.makeConstraints { make in
+            make.top.equalToSuperview()
+            make.leading.equalToSuperview()
+            make.trailing.equalToSuperview()
+            make.bottom.equalToSuperview()
+        }
```

Comment:
You can make this `make.edges.equalToSuperView()` and all four constraints will be applied

Reply:
ahh nice, I was hoping they had something like that.